ATR/main.yaml

```
@@ -0,0 +1,12 @@
+sources:
+  track:
+    description: ATR track
+    driver: opendap
+    args:
+      urlpath: https://observations.mpimet.mpg.de/repository/opendap/16b222a4-0ce5-42ac-95f6-3625add76175/entry
+      auth: null
+      chunks: {}
+      engine: netcdf4
+    metadata:
+      tags:
+        - tracks
```

ATR uses `tracks` and HALO uses `track` as a tag. I have no particular preference, but `track` seems to be consistent to the other occurrences.
```suggestion
        - track
```